react/features/video-status-label/components/VideoStatusLabel.js

```diff
@@ -136,54 +144,41 @@ export class VideoStatusLabel extends Component {
             = `${baseClasses} ${filmstrip} ${remoteVideosVisible} ${opening}`;
 
         return (
-            <div
```

Comment:
How about if I work on that once this pull request is settled? And I mean like the day of or the day after it gets merged. I agree but my only concern is that change strays a bit much from the pull request intent, but I'm okay with that if you're okay with that. I think what needs to happen is the label needs to get pulled into the video-quality directory and then the logic changed for determining HD/SD.